demisto_sdk/commands/format/tests/test_formatting_readme_test.py

```diff
@@ -44,8 +55,9 @@ class TestReadmeFormat:
     @pytest.mark.parametrize('regex_relative_url,new_url,expected_link',
                              ((['[invalid relative 2]', 'www.relative2.com', True], 'https://new.com',
                                '[invalid relative 2](https://new.com)'), (
-                              ['<a href="www.hreftesting.com"', 'www.hreftesting.com', False], 'https://new.com',
-                              '<a href="https://new.com"'),
+                                      ['<a href="www.hreftesting.com"', 'www.hreftesting.com', False],
+                                      'https://new.com',
+                                      '<a href="https://new.com"'),
```

Comment:
seems like a spacing-only change. if it's so, please revert to keep the git-blame clean :)